test/factory/common_factory_tests.dart

```diff
@@ -82,7 +85,7 @@ void commonFactoryTests(ReactComponentFactoryProxy factory) {
   });
 }
 
-void domEventHandlerWrappingTests(ReactDomComponentFactoryProxy factory) {
+void domEventHandlerWrappingTests(ReactComponentFactoryProxy factory) {
```

Comment:
I went back and forth on this line. I was pulling test code from a [commit Greg made](https://github.com/greglittlefield-wf/react-dart/commit/273b9152bff3455245e26f846d8fb07a915a1b09#diff-7a382a75d4df367a3f5df34d837d9227) a month ago (which was the source of the function in `js_interop_helpers.dart`) and came across this line. Since it was a month ago it seemed like a recent deliberate change in the type so I brought it over. Did I read the commit wrong or is still `ReactDomComponentFactoryProxy` preferred?